### PR TITLE
Exporting a Model from PyTorch to ONNX tutorial - Docathon

### LINKED ISSUE #2902: Test the Exporting a Model from PyTorch to ONNX tutorial
Test the following tutorial:
 https://pytorch.org/tutorials/advanced/super_resolution_with_onnxruntime.html
 **Follow these steps:**
1. Fork the pytorch/tutorials repo.
2. Switch to a new branch.
3. Run the tutorial as a python script:
```
python3 <path-to-tutorial>
```
4. Save the output into a .txt file. Did you see any errors? Add them to the output.
5. Run the tutorial in the Google Colab and save the outputs into a .txt file. Did you see any issues? Take screenshots of the issues.
6. Did the tutorial make sense? Can you suggest any improvements? Past all the outputs, screenshots, and suggestions for improvements in the issue and ping one of the reviewers. If your suggestion is approved, please submit a PR with the fix.

cc @xadupre @svekars @kit1980 @brycebortree

### PR DESCRIPTION
**Pull request by diningeachox**
Fixes #2902 

## Description
Adds a timing script to the tutorial page to compare inference times between pytorch and ONNX models.
Fixed one of the external links at the bottom of the page.
Displays low-res and super-res images next to each other so readers can see a comparison.

## Checklist

- [x] The issue that is being fixed is referred in the description (see above "Fixes #ISSUE_NUMBER")
- [x] Only one issue is addressed in this pull request
- [x] Labels from the issue that this PR is fixing are added to this pull request
- [x] No unnecessary issues are included into this pull request.

cc @xadupre @sekyondaMeta @svekars @kit1980 @brycebortree, @titaiwangms and @thiagocrepaldi

### CODE REVIEW

**Comment by svekars** [comment]
@xadupre, @titaiwangms  @thiagocrepaldi

**Review by xadupre — APPROVED** [review]

**Review by titaiwangms — APPROVED** [review]
Thanks!

File: advanced_source/super_resolution_with_onnxruntime.py [hunk 1]
```diff
@@ -301,8 +327,12 @@ def to_numpy(tensor):
 # Save the image, we will compare this with the output image from mobile device
 final_img.save("./_static/img/cat_superres_with_ort.jpg")
 
+# Save resized original image (without super-resolution)
+img = transforms.Resize([img_out_y.size[0], img_out_y.size[1]])(img)
+img.save("cat_resized.jpg")
 
 ######################################################################
+# .. figure:: /_static/img/cat_resized.jpg
```
Comment (svekars) [inline advanced_source/super_resolution_with_onnxruntime.py:335]: So now we have this image displayed twice: https://docs-preview.pytorch.org/pytorch/tutorials/2935/advanced/super_resolution_with_onnxruntime.html 

If you are comparing them, add some text in between the two images?

Comment (diningeachox) [reply advanced_source/super_resolution_with_onnxruntime.py:335]: I've added captions to each of the two images.

File: advanced_source/super_resolution_with_onnxruntime.py [hunk 2]
```diff
@@ -301,10 +327,18 @@ def to_numpy(tensor):
...
```
Comment (svekars) [inline advanced_source/super_resolution_with_onnxruntime.py:335]:
```suggestion
# Here is the comparison between the two images: 
#
# .. figure:: /_static/img/cat_resized.jpg
```

File: advanced_source/super_resolution_with_onnxruntime.py [hunk 3]
```diff
@@ -301,10 +327,18 @@ def to_numpy(tensor):
...
+#
+# Low-res image
```
Comment (svekars) [inline advanced_source/super_resolution_with_onnxruntime.py:337]:
```suggestion
# Low-resolution image:
```

**Review by svekars — APPROVED** [review]

**Pull request merged**